index.html

```diff
@@ -1,108 +1,280 @@
-                <ul>
-                    <li>JavaScript</li>
-                    <li>Ruby</li>
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-            </div>
-            <div>
-                <div class="technologies-item">
-                    <div class="technologies-title">Frameworks</div>
-                    <ul>
-                        <li>BootStrap</li>
-                        <li>Ruby on Rails</li>
-                        <li>RPSec</li>
-                        <li>CapyBase</li>
-                        <li>Selenium</li>
-                    </ul>
-                </div>
-            </div>
-            <div>
-                <div class="technologies-item">
-                    <div class="technologies-title">Skills</div>
-                    <ul>
-                        <li>Codekit</li>
-                        <li>GitHub</li>
-                        <li>CodePEn</li>
-                        <li>GitLab</li>
-                        <li>Terminal</li>
-                    </ul>
-                </div>
+      <div class="divider"></div>
+      <!-- TECHNLOGIES -->
+      <section class="technologies">
+        <div class="technologies-item">
+          <h2 class="technologies-title">Languages</h2>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div>
+          <div class="technologies-item">
+            <div class="technologies-title">Frameworks</div>
+            <ul>
+              <li>BootStrap</li>
+              <li>Ruby on Rails</li>
+              <li>RPSec</li>
+              <li>CapyBase</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div>
+          <div class="technologies-item">
+            <div class="technologies-title">Skills</div>
+            <ul>
+              <li>Codekit</li>
+              <li>GitHub</li>
+              <li>CodePEn</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <img class="second-img" src="./circle_header.svg" alt="circle 2 header"/>
+      <div class="divider"></div>
+
+      <!-- PAGE BREAK -->
+      <section class="page-break">
+        <div>
+          <p>Work &darr;</p>
+        </div>
+        <div class="social-links">
+          <a href="https://github.com/Marlyn_Mayienga"
+            ><i class="fab fa-github"></i
+          ></a>
+          <a href="https://www.linkedin.com/in/marlyn-mayienga/"
+            ><i class="fab fa-linkedin"></i
+          ></a>
+          <a href="https://www.instagram.com/merl.__/?hl=en"
+            ><i class="fas fa-cat"></i
+          ></a>
+          <a href="https://twitter.com/M_ayienga"
+            ><i class="fab fa-twitter"></i
+          ></a>
+          <a href="marlymayienga@gmail.com"><i class="fas fa-envelope"></i></a>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)